src/d/alexa-openwebif/source/intents/remotecontrol.d

```diff
@@ -63,9 +66,10 @@ final class IntentRCPrevious : BaseIntent
 	///
 	override AlexaResult onIntent(AlexaEvent, AlexaContext)
 	{
-		auto result = doRCIntent("Previous",apiClient,this);	
+		AlexaResult result;
 		result.response.card.title = getText(TextId.RCPreviousCardTitle);
 		result.response.card.content = getText(TextId.RCPreviousCardContent);
```

Comment:
!? don't get it. This is neccessary because the error functions sets the card title and content. So I need to define `AlexaResult result` first, set the "normal" card titles and make the api call with the nested error check. On error the `doRCIntent()` function returns a `AlexaResult` object with Error Card Title and Content. In case of no error it only returns the SSML.